app/questionnaire/rules.py

```diff
@@ -67,14 +67,42 @@ def evaluate_condition(condition, answer_value, match_value):
     comparison_operators = {
         'equals': lambda answer_value, match_value: answer_value == match_value,
         'not equals': lambda answer_value, match_value: answer_value != match_value,
-        'contains': lambda answer_value, match_value: isinstance(answer_value, list) and match_value in answer_value,
-        'not contains': lambda answer_value, match_value: isinstance(answer_value, list) and match_value not in answer_value,
+        'contains':
+            lambda answer_value, match_value: isinstance(answer_value, list)
+            and match_value in answer_value,
+        'all of':
```

Comment:
When launched "All of" fails if other items selected too.

e.g. 
	"condition": "all of",
	"values": ["Box1", "Box2", "Box3"]
passes if Box1, Box2, Box3 selected but
fails if Box1, Box2, Box3, Box4 selected